### PR TITLE
Feature/68 on mobile replace vertical scrolling with horizontal scrolling

### PR DESCRIPTION
**Pull request by tmueller**

### CODE REVIEW

**Comment by tmueller** [comment]
Please test thoroughly. Mobile, Desktop. Different browsers. Dark mode, light mode. Thanks :)

**Comment by tmueller** [comment]
**Remixicons loaded twice, if cache is deactivated**
Cannot reproduce, only happens if cache is off. Unlikely on prod.

**Third dot is not animated, when on >60Hz Screens**
Should work for screen up to 240Hz now

**Navlinks need a name attribute**
Please test, whether lighthouse is happy now.

**Comment by kirkone** [comment]
Lighthouse is still not happy with the links.
More information about the solution:
https://dequeuniversity.com/rules/axe/4.8/link-name

image

**Comment by tmueller** [comment]
Let's hope we obey lighthouse enough now.

**Comment by kirkone** [comment]
> Let's hope we obey lighthouse enough now.

image

File: src/Cards/src/components/share-page.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+            <DownloadLink
+                downloadUrl={downloadUrl}
+                name={name}
+                className={styles.githubMargin}
+            />
+
+            <DetailLine>
+                <DetailLink
+                    icon={GithubFillIcon}
+                    caption="Open Source on github"
```
Comment (kirkone) [inline src/Cards/src/components/share-page.tsx:48]:
```suggestion
                    caption="Documentation and code on GitHub"

```

File: src/Cards/src/components/share-page.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                    icon={ErrorIcon}
+                    caption="Error loading contact data"
+                />
+            </DetailLine>
+        ),
+        Loaded: ({ url }) => (
+            <DetailLine className={className}>
+                <DetailLink
+                    icon={DownloadIcon}
+                    caption="Download contact"
```
Comment (kirkone) [inline src/Cards/src/components/share-page.tsx:89]:
```suggestion
                    caption="Download this snappy.card as vCard"

```

File: src/Cards/src/components/share-page.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                                })
+                            )
+                        })
+                    )}
+                    caption={pipe(
+                        shareMode,
+                        matchShareMode({
+                            none: constant(""),
+                            clipboard: constant("Copy snappy card link"),
+                            share: constant("Share snappy card link"),
```
Comment (kirkone) [inline src/Cards/src/components/share-page.tsx:154]:
```suggestion
                            clipboard: constant("Copy snappy.card link"),
                            share: constant("Share snappy.card link"),

```

**Review by kirkone — APPROVED** [review]
Looks incredibly good

**Review by kirkone — APPROVED** [review]
🥁

**Pull request merged**